cms/use_cases/case.py

```diff
@@ -74,5 +74,12 @@ def view_case(repo, case_id):
         if case is None:
             raise Exception("Case not found")
         return ResponseSuccess(case)
+    except Exception as exc:
+        return ResponseFailure(ResponseTypes.SYSTEM_ERROR, exc)
+
+def case_list_from_beneficiary_id(repo, beneficiary_id):
+    try:
+        case = repo.case_list_with_beneficiaryid(beneficiary_id)
```

Comment:
name the variables which gives actual sense.  You can name it case_list

Reply:
when I name it case_list it gives error as there is already a method called case_list() in case.py file.

Reply:
you can rename it to different thing like clist or anything else ?

Reply:
Ok